qa/smoke-test-multinode/src/test/resources/rest-api-spec/test/smoke_test_multinode/20_tsdb_consistency.yml

```diff
@@ -68,8 +68,8 @@ setup:
 ---
 "index with replicas and shards is green":
   - skip:
-      version: " - 7.99.99"
-      reason: introduced in 8.0.0 to be backported to 7.16.0
+      version: " - 8.0.99"
+      reason: introduced in 8.1.0
```

Comment:
Indexing should work in 8.0 too. Is there a specific reason you have in mind that would make this test fail in 8.0?

Also, yes. This will not be backported to 7.16

Reply:
maybe it is an old code